### PR TITLE
feat: Update `Makefile` to use Podman instead of Docker

### PR DESCRIPTION
**Pull request by DMunkei**

### CODE REVIEW

**Review by qasimwarraich — CHANGES_REQUESTED** [review]
Please rebase your branch against main, or drop the two extraneous commits from myself.

**Comment by DMunkei** [comment]
Dropped both of the commits. Look good now?

File: README.md [hunk 1]
```diff
@@ -92,33 +92,35 @@ Change directory into the freshly cloned repo.
...
+- [Podman](https://podman.io/)
 - make (optional)
 
 Instructions:
 
-Option 1: Pull down the image from [Docker Hub](https://hub.docker.com/r/qasimwarraich/cli-tutor)
+Option 1: Pull down the image from [Github Container Registry](https://ghcr.io/qasimwarraich/cli-tutor:latest)
 
-`docker pull qasimwarraich/cli-tutor`
+`podman pull qasimwarraich/cli-tutor`
```
Comment (qasimwarraich) [inline README.md]: This wont work out of the box, the registry name is required.

**Review by qasimwarraich — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (DMunkei) [reply README.md]: Isn't this already addressed in the other commit? The source was changed to ghcr.

Comment (qasimwarraich) [reply README.md]: The link for the pull command needs to have the full link for the registry, just try it locally and you will see what I am talking about.

Comment (DMunkei) [reply README.md]: I'll try it when I'm on the pc. I tried the commands before pushing them. I probably already had the registry locally pulled.

Comment (DMunkei) [reply README.md]: Saw my mistake. New commit should fix it. As an aside, the remaining `docker` commands in the `README` file are addressed in the other PR. Didn't want to go fixing them here also.

**Comment by DMunkei** [comment]
I also don't see any reason why we can't add a pull target in the `Makefile`?
```make
podmanpull:
	podman pull ghcr.io/qasimwarraich/cli-tutor:latest
```
I can tack this on if you want.

File: README.md (hunk 1 above)
Comment (qasimwarraich) [reply README.md]: > Saw my mistake. New commit should fix it. As an aside, the remaining `docker` commands in the `README` file are addressed in the other PR. Didn't want to go fixing them here also.

Nice! Now just fix the typo in your commit message, capitalise the first word of all the messages, and we are pretty much done here.

**Comment by qasimwarraich** [comment]
> I also don't see any reason why we can't add a pull target in the `Makefile`?
> 
> ```makefile
> podmanpull:
> 	podman pull ghcr.io/qasimwarraich/cli-tutor:latest
> ```
> 
> I can tack this on if you want.

Sure

**Comment by DMunkei** [comment]
Added it. Anything else?

**Comment by qasimwarraich** [comment]
> Added it. Anything else?

Just fix the typos and capitalise the first letter in the message part of the commit messages.
https://github.com/qasimwarraich/cli-tutor/pull/5#discussion_r1541052703

**Comment by DMunkei** [comment]
Done. Didn't catch the typo. woops.

**Comment by DMunkei** [comment]
Seems like fixing the conflict here in github made an additional commit. Don't merge it in yet.

**Review by qasimwarraich — APPROVED** [review]

**Pull request merged**